app/assets/javascripts/app/form-field-format.js

```diff
@@ -13,6 +14,7 @@ function formatForm() {
     ['.mfa', new NumericFormatter()],
     ['.mortgage', new NumericFormatter()],
     ['.phone', new PhoneFormatter()],
+    ['.pkey', new PersonalKeyFormatter()],
```

Comment:
this is a tiny thing, but when I was recently renaming `recovery_code` to `personal_key` everywhere, things like this would slip through my global find searches. Any chance we can name it `.personal-key` instead?